tutorials/basic-topics/sorting/cyclic-sort.md

```diff
@@ -11,4 +11,97 @@ keywords:
   - cyclic sort
 ---
 
-<TutorialAuthors names="@TBC"/>
+<TutorialAuthors names="@prishit55"/>
+
+# Overview
+
+Cycle-Sort is an in-place, unstable-sorting algorithm.
+* `in-place` : An in-place algorithm transforms input without using any other auxiliary data structure. As the algorithm executes the input is overwritten by the output.
+* `unstable` : Unstable sorting algorithm don't preserve the relative order of equal elements while sorting.
+
+
+Cycle sort algorithm factors the permutation to be sorted into number of cycles, which are individually rotated to give desired sorted result.
+
+## Algorithm
```

Comment:
This part should follow what you make in the diagram.

Reply:
Unless the correct element reaches the starting index variable ' i ' will not get updated. That depicts one cycle this is what I am trying to show in the diagram.
I'll write this part somewhere in the explanation part.